backend/services/implementations/__tests__/classService.test.ts

```diff
@@ -174,8 +176,19 @@ describe("mongo classService", (): void => {
     }).rejects.toThrowError(`Class with id ${notFoundId} not found`);
   });
 
+  it("archive class", async () => {
+    // add test class
+    const classObj = await ClassModel.create(mockTestClass);
+
+    // execute
+    const res = await classService.archiveClass(classObj.id);
+
+    // assert
+    expect(res.isActive).toEqual(false);
+  });
```

Comment:
can we add 2 test cases? 
1. when the id is not found 
2. when the class is already archived